test/test_alpha_conversion.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import absolute_import, division, print_function
+
+from collections import OrderedDict
+
+import pytest
+
+import funsor.ops as ops
+from funsor.domains import bint, reals
+from funsor.interpreter import gensym, interpretation, _GENERIC_SUBS
+from funsor.terms import Independent, Lambda, Variable, reflect
+from funsor.testing import assert_close, check_funsor, random_tensor
+
+
+# TODO remove this when removing eager_subs methods
+xfail_if_old_subs = pytest.mark.xfail(not _GENERIC_SUBS, reason="fails w/ old subs")
```

Comment:
I've marked these new tests exercising alpha-renaming with a conditional `xfail`.  I will remove this as part of addressing #149